persistence/registry/ginadll/persist-via-ginadll-registry-key.yml

```diff
@@ -9,11 +9,13 @@ rule:
     examples:
       - Practical Malware Analysis Lab 11-01.exe_:0x401000
   features:
-    - and:
-      - or:
+    - or:
```

Comment:
I think there's a similar problem to the appinit rule here, i.e. that the `create process` block misses the `GinaDLL` string feature

Reply:
Thank you for your comments @mr-tz! The `GinaDLL` string is already part of the Regex pattern `/REG ADD[\s\"\\a-z_]{5,20}Software\\Microsoft\\Windows NT\\CurrentVersion\\Winlogon\\GinaDLL/i`. I think I wanted to do the same for the AppInit rule but missed it somehow. Nevertheless, let's keep it consistent and I will change it in the upcoming commit.